server/src/services/proposal.services.js

```diff
@@ -748,47 +862,49 @@ export const createProposalRequest = async (
           }
         }
         resolve({
-            id: this.lastID,
-            student_id: student_id,
-            teacher_id: teacher_id,
-            co_supervisors_ids: co_supervisors_ids,
-            title: title,
-            type: type,
-            description: description,
-            notes: notes,
-            status: "submitted",
+          id: this.lastID,
+          student_id: student_id,
+          teacher_id: teacher_id,
+          co_supervisors_ids: co_supervisors_ids,
+          title: title,
+          type: type,
+          description: description,
+          notes: notes,
+          status: "submitted",
         });
       }
     );
   });
 };
 
-export const getProposalRequestsFromDB = async () => {
+export const getProposalRequestsFromDB = async (teacherId = undefined) => {
```

Comment:
Is this `teacherId` supposed to be here? If yes why?

Reply:
Thank you for your observation. No it was just for testing. I'll remove it.